### PR TITLE
Finnish language update

### PR DESCRIPTION
**Pull request by mediaclinic**
Updated Finnish language files.

### CODE REVIEW

**Comment by LukeTowers** [comment]
You have a syntax error: https://travis-ci.org/octobercms/october/jobs/501130739#L680

**Comment by mediaclinic** [comment]
Missed ], and added also 'scopes'

**Comment by LukeTowers** [comment]
@w20k could you look into the failing test?

**Comment by w20k** [comment]
@LukeTowers - sure, I even know, probably, why.

File: modules/system/lang/fi/lang.php [hunk 1]
```diff
@@ -380,8 +425,8 @@
         'log_theme' => 'Kirjaa teeman muutokset',
         'log_theme_comment' => 'Kun muutos on tehty teemaan käyttäen hallintapaneelia.',
     ],
-    'media' => [
-        'invalid_path' => "Kelvoton tiedostopolku määritetty: ':path'.",
-        'folder_size_items' => 'kohteita',
+        'media' => [
```
Comment (w20k) [inline modules/system/lang/fi/lang.php]: @mediaclinic please remove tab character.

Comment (mediaclinic) [reply modules/system/lang/fi/lang.php]: Removed tabs and one comma.

File: modules/backend/lang/fi/lang.php [hunk 2]
```diff
@@ -2,188 +2,188 @@
...
+            'label' => 'Token on virheellinen'
         ]
     ],
     'partial' => [
-        'not_found_name' => "Osaa ':name' ei löydy."
+        'not_found_name' => "Osiota ':name' ei löydy."
     ],
     'account' => [
-        'signed_in_as' => 'Kirjautunut käyttäjänä :full_name',
+	    'signed_in_as' => 'Kirjauduit nimellä :full_name',
```
Comment (LukeTowers) [inline modules/backend/lang/fi/lang.php]: You have a tab here, remove it please

File: modules/system/lang/fi/lang.php [hunk 3]
```diff
@@ -380,8 +425,8 @@
         'log_theme' => 'Kirjaa teeman muutokset',
         'log_theme_comment' => 'Kun muutos on tehty teemaan käyttäen hallintapaneelia.',
     ],
-    'media' => [
-        'invalid_path' => "Kelvoton tiedostopolku määritetty: ':path'.",
-        'folder_size_items' => 'kohteita',
-    ],
+        'media' => [
```
Comment (LukeTowers) [inline modules/system/lang/fi/lang.php]: You have an extra indentation here, please remove.

**Comment by w20k** [comment]
@mediaclinic if you're still interested in this PR. Please fix tabs and spaces, and could be merged.

**Comment by mediaclinic** [comment]
Should be fixed now.

On 3. May, 2019 at 18:16:07 , Denis Denisov (notifications@github.com) wrote:

@mediaclinic if you're still interested in this PR. Please fix tabs and spaces, and could be merged.

**Pull request merged**